### PR TITLE
[fix/trivial] Fix compile warning of g_param_spec_string in gsttensormerge.c

### PR DESCRIPTION
**Pull request by anyj0527**
- Set default string value as argument of `g_param_spec_string()`

Signed-off-by: Yongjoo Ahn <yongjoo1.ahn@samsung.com>

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

File: gst/nnstreamer/tensor_merge/gsttensormerge.c [hunk 1]
```diff
@@ -155,7 +160,7 @@ gst_tensor_merge_class_init (GstTensorMergeClass * klass)
   g_object_class_install_property (gobject_class, PROP_MODE,
       g_param_spec_string ("mode", "Mode",
           "Tensor Merge mode. Currently, `linear` is available only.",
-          GTT_LINEAR, G_PARAM_READWRITE));
+          DEFAULT_PROP_MODE, G_PARAM_READWRITE));
```
Comment (dongju-chae) [inline gst/nnstreamer/tensor_merge/gsttensormerge.c:163]: Nice catch :)

**Review by dongju-chae — APPROVED** [review]
LGTM

File: gst/nnstreamer/tensor_merge/gsttensormerge.c [hunk 2]
```diff
@@ -91,6 +91,11 @@ enum
   PROP_SILENT,
 };
 
+/**
+ * @brief Default mode
+ */
+#define DEFAULT_PROP_MODE "linear"
```
Comment (wooksong) [inline gst/nnstreamer/tensor_merge/gsttensormerge.c:97]: How about using ```gst_tensor_merge_mode_string```?

Comment (anyj0527) [reply gst/nnstreamer/tensor_merge/gsttensormerge.c]: Thanks! I reused the function.

**Review by wooksong — APPROVED** [review]
LGTM

**Pull request merged**